pkg/kubelet/kuberuntime/security_context_test.go

```diff
@@ -105,7 +105,8 @@ func TestVerifyRunAsNonRoot(t *testing.T) {
 		},
 	} {
 		pod.Spec.Containers[0].SecurityContext = test.sc
-		err := verifyRunAsNonRoot(pod, &pod.Spec.Containers[0], int64(0))
+		uid := int64(0)
+		err := verifyRunAsNonRoot(pod, &pod.Spec.Containers[0], &uid, "")
```

Comment:
add also test with verifyRunAsNonRoot(pod, &pod.Spec.Containers[0], nil, "test") ?

Reply:
We have a one test already and it should be enough to make this merged. In order to not block this PR, I'm going to add a test next week as a follow-up. @liggitt didn't object against this.